adax/__init__.py

```diff
@@ -31,11 +34,26 @@ def __init__(self, account_id, password, websession):
         self._write_task = None
         self._pending_writes = {"rooms": []}
 
+    async def get_homes(self):
```

Comment:
I think it would be better to not do self.update() in all these functions. Should be possible to get self._homes without forcing an update.
(Should not had self.update() in get_rooms either)

Reply:
I didn't wanted to such refactor. Imho, the cleanest would be to have functions which are 1-on-1 with the API endpoints.